### PR TITLE
Stop travis runs by removing config file and update documentation

### PR DESCRIPTION
**Pull request by KKoukiou**
We run our tests reliably on testing farm and we stopped our travis subscription.

### CODE REVIEW

**Review by M4rtinK — APPROVED** [review]
Looks good to me. :)

File: plans/testing-farm.fmf [hunk 1]
```diff
@@ -37,7 +37,7 @@ discover:
         trap "cp -rv data/logs/ ${TMT_TEST_DATA}/" EXIT INT QUIT PIPE
 
         cd /root/kickstart-tests
-        scripts/run_travis.sh
+        scripts/run.sh
```
Comment (jkonecny12) [inline plans/testing-farm.fmf:40]: We should use a more specific name. I'm afraid of people running this to run kickstart-tests.
Maybe `run-in-tmt` or `run-ci`?

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

File: plans/testing-farm.fmf (hunk 1 above)
Comment (KKoukiou) [reply plans/testing-farm.fmf]: Good point.

**Pull request merged**